### DISCUSSION TITLE
totale pagamenti errati

### DISCUSSION THREAD

**Discussion by Annaanto** [fattureincloud/api#77] [Q&A] [answered]
Buongiorno,
sto mandando questa semplice fattura tramite API

{
	"data" : 
	{
		"currency" : 
		{
			"exchange_rate" : "1.00000",
			"id" : "EUR",
			"symbol" : "€"
		},
		"date" : "2022-09-15",
		"entity" : 
		{
			"id" : 45556032
		},
		"item_list" : 
		[
			{
				"apply_withholding_tax" : true,
				"category" : "803001",
				"description" : "descrizione test",
				"gross_price" : 1171.2,
				"net_price" : 960,
				"not_taxable" : false,
				"qty" : 1,
				"vat" : 
				{
					"id" : 2796374,
					"value" : 22
				}
			}
		],
		"number" : 242,
		"payment_metod" : 
		{
			"Name" : "MP05",
			"bank_iban" : "xxx",
			"bank_name" : "xxx"
		},
		"payments_list" : 
		[
			{
				"amount" : 1171.2,
				"due_date" : "22/09/2022"
			}
		],
		"type" : "invoice",
		"year" : 2022
	}
}

Ma mi ritorna l'errore
**{"message":"Il totale dei pagamenti non corrisponde al totale da pagare.","validation_result":null}**

L'ID VAT è corretto perchè ho fatto una get List Vat Types e ho anche controllato controllato i totali con la chiamata Get New Issued Document Totals come suggerito in un altro post analogo e l'importo restituito è 1171.2

Dove sbaglio?

Grazie,
Anna

**Comment by valmoz** [comment, marked as answer]
Buongiorno Anna,
ha già letto la [guida dedicata ai totali](https://developers.fattureincloud.it/docs/guides/invoice-totals)?

Grazie
Mauro

**Reply by Annaanto** [reply]
Si. Usando Get New Issued Document Totals ottengo lo stesso totale.
Anna

**Reply by valmoz** [reply]
Buongiorno,
dal body della sua richiesta vedo che sta utilizzando "item_list" invece di "items_list" come nome del campo, quindi l'API non è in grado di identificare gli items e ottiene un totale pari a 0.

Le consiglio quindi di correggere il nome del campo.
Ci faccia sapere se in questo modo riesce a risolvere.

Grazie
Mauro

**Reply by Annaanto** [reply]
Ho risolto.
Grazie 
Anna

**Reply by valmoz** [reply]
Ottimo!

Mauro
